packages/contracts/contracts/optimistic-ethereum/OVM/predeploys/OVM_DeployerWhitelist.sol

```diff
@@ -185,28 +137,10 @@ contract OVM_DeployerWhitelist is iOVM_DeployerWhitelist {
             bool _allowed
```

Comment:
I think we're moving away from named return values, good time to get that in here. 
```suggestion
            bool
```